content/about/duje/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: About - Duje Schröder
+weight: 10
+---
+
+<div class="text-center mb-5">
+    <img
+        src="https://avatars.githubusercontent.com/u/23352538?v=4"
+        width="150"
+        class="rounded-circle mt-3"
+    />
+    <h3 class="m-3">Duje Schröder</h3>
+    <p>Assistant Coordinator</p>
+</div>
+
+<div class="text-center">
+   _,--',     _,_._.--._______
+ .--.--';_'-. ',_";_      _.,-'
+.'--'.  _.'    {`'-;_ .-...'
+      '-:_      )  / `' '=.
+        ) >     {_/     /.)..
+        |/               `..; .'
+</div>
+
+
+End? No, the journey doesn't end here. MapLibre is another path. One that we
+all must take. On 2020-12-08 22:42:56 UTC MapLibre restarted mapping with its
+mission to provide freedom and independence. Eventually, this is what everybody
```

Comment:
```suggestion
mission to provide freedom and independence. In the end, this is what everybody
```

Reply:
https://dictionary.cambridge.org/dictionary/english/eventually 🤔

"In the end" sounds final as in at the end of a period, whereas "eventually" feels more like "some day (in the future)" and less opinionated or like a statement.